common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch6/ElasticSearchClient.scala

```diff
@@ -53,6 +53,11 @@ trait ElasticSearchClient {
     }
   }
 
+  def healthCheck(): Future[Boolean] = {
+    val request = search(imagesAlias) limit 0
+    executeAndLog(request, "Health check").map { _ => true}.recover { case _ => false}
+  }
+
```

Comment:
maybe limit 1 ?

Reply:
Discussed over chat: `limit` here is the maximum to return, as we're primarily interested in knowing if we can query ES (rather than ES's response for an image) `limit 0` should be fine.

This PR is also mainly moving existing code around and doesn't reason about it's quality too much. That is, let's change this in it's own PR if needed.